client/frame-runner.js

```diff
@@ -29,7 +52,10 @@ document.addEventListener('DOMContentLoaded', function() {
 
   document.__runTests = function runTests(tests = []) {
     /* eslint-disable no-unused-vars */
-    const editor = { getValue() { return source; } };
+    const editor = {
+      getValue() { return source; },
+      getOriginalCode() { return originalCode; }
+    };
```

Comment:
The tests that require access to user code should use `getUserInput` see: https://github.com/freecodecamp/freecodecamp/blob/staging/seed/challenges/03-front-end-libraries/react.json#L65